scipy/sparse/linalg/_isolve/iterative.py

```diff
@@ -524,11 +524,11 @@ def cgs(A, b, x0=None, *, rtol=1e-5, atol=0., maxiter=None, M=None, callback=Non
     for iteration in range(maxiter):
         rnorm = np.linalg.norm(r)
         if rnorm < atol:  # Are we done?
-            return postprocess(x), 0
+            return x, 0
 
         rho_cur = dotprod(rtilde, r)
         if np.abs(rho_cur) < rhotol:  # Breakdown case
-            return postprocess, -10
```

Comment:
This looks like a bug that was never hit, the intent here clearly was to return `postprocess(x0`